test/csuite/timestamp_abort/main.c

```diff
@@ -353,6 +355,8 @@ thread_run(void *arg)
      */
     printf("Thread %" PRIu32 " starts at %" PRIu64 "\n", td->info, td->start);
     active_ts = 0;
+    if (stress)
+        session->verbose_private = private;
```

Comment:
To be safe, `private` should be initialized before setting `verbose_private`, so there's no chance of a verbose message with a garbage value.

Reply:
Fixed